CMakeLists.txt

```diff
@@ -810,7 +814,7 @@ if(HPX_COMPILER_WARNINGS)
       hpx_add_compile_flag_if_available(-Werror=uninitialized)
     endif()
 
-    #hpx_add_compile_flag_if_available(-Werror=missing-field-initializers)
+    hpx_add_compile_flag_if_available(-Wno-missing-field-initializers)
```

Comment:
Why was this warning disabled? It seems it was considered an error before. In case this is actually needed, it seems it should be moved out of the warnings-as-errors section.